Framework/CurveFitting/src/Algorithms/SplineSmoothing.cpp

```diff
@@ -303,10 +324,10 @@ void SplineSmoothing::addSmoothingPoints(const std::set<int> &points,
  * @param row :: The row of spectra to use
  */
 void SplineSmoothing::selectSmoothingPoints(
-    MatrixWorkspace_const_sptr inputWorkspace, size_t row) {
+    MatrixWorkspace_const_sptr inputWorkspace, const size_t row) {
```

Comment:
`selectSmoothingPoints(*myWS, row);` doesn't look to bad to me (one character more, saving many somewhere else).